### PR TITLE
Refactor using import-cost module

### PR DESCRIPTION
**Pull request by gregorym**
Use [import-cost](https://github.com/wix/import-cost/tree/master/packages/import-cost) to calculate import sizes. I removed all the unnecessary stuff and did a bit of function renaming.

### CODE REVIEW

**Comment by gregorym** [comment]
Takes care of this issue #2

File: styles/atom-import-cost.less [hunk 1]
```diff
@@ -7,5 +7,5 @@
   opacity: 0.5;
   margin-left: 10px;
   position: relative;
-  top: -(@component-line-height - (@component-padding/2));
+  top: -@component-line-height;
```
Comment (reznord) [inline styles/atom-import-cost.less:10]: you have to leave the padding as it is. Since it will disturb the position of the `size` from the top by 2px.

Comment (gregorym) [reply styles/atom-import-cost.less:10]: it looks pretty wells centered for me.
screen shot 2017-08-10 at 10 38 35 am

Comment (reznord) [reply styles/atom-import-cost.less:10]: hmm! Probably because I've changed the font! (but not sure how that can effect)

Comment (gregorym) [reply styles/atom-import-cost.less:10]: I also use a custom font. I just switched to the default font and it looks properly aligned.

Comment (reznord) [reply styles/atom-import-cost.less:10]: yeah, in custom font, it breaks a bit bad :P (but its fine) people hardly change it ;)

**Review by reznord — APPROVED** [review]
LGTM!

**Comment by reznord** [comment]
@siddharthkp this changes looks good to me. Waiting for your review & merge

**Comment by siddharthkp** [comment]
Whoops, this completely stopped working for me. Reverted it for now: 30654bc5e79e4c0d550de46a2c0c36dc9e01e279

Let me dig deeper into what happened.

**Review by reznord — COMMENTED** [review]
There are a few things: 

1) loader is not working (not sure why)
2) This works only when the package is available in `node_modules` folder.

**Comment by gregorym** [comment]
How can I repro this issue?

**Comment by siddharthkp** [comment]
I didn't get time to debug it properly after that 😢 

Let me try to get some help with this

**Comment by siddharthkp** [comment]
This is the error that I was getting. 

```
Failed to require the main module of 'atom-import-cost' because it requires one or more incompatible native modules (binding, binding, binding, binding, binding, binding).
Run `apm rebuild` in the package directory and restart Atom to resolve.
```

I've published it again as `atom-import-cost-beta`, can you try if this works for you?

**Comment by gregorym** [comment]
Ok, i will take a look.

**Comment by gregorym** [comment]
`import-cost` has a native dependency that needs to be rebuilt by Atom.. 😢 it works after that.
I'll try to see if there is a work-around.

**Comment by gregorym** [comment]
One of the dependency of `import-cost` depends on `fsevents` which causes the extension to not work after it is installed.

The work around is to rebuild the failing dependency by clicking on the red bug icon at the bottom right of Atom. it is not a great experience but that's the best we can do for now...

Examples:
- https://github.com/yongkangchen/remote-sync/issues/434
- https://github.com/TypeStrong/atom-typescript/issues/583

**Comment by siddharthkp** [comment]
That doesn't sound good 😞 

I've been playing with the idea of keeping the installs and bundling on a remote server that uses the same package and talk to it via an API.
We have add an optimisation on top of this which caches the size of package@version on the client.

Cons: Need internet access to get the size of a package@version you are using for the first time.

**Comment by reznord** [comment]
I was just going through this: https://github.com/electron/electron/blob/master/docs/tutorial/using-native-node-modules.md

Will dig into it more and drop here if I come across something useful.

**Pull request merged**